source/neuropod/bindings/java/src/main/native/com_uber_neuropod_Neuropod.h

```diff
@@ -22,13 +22,6 @@ limitations under the License.
 #ifdef __cplusplus
 extern "C" {
 #endif
-/*
- * Class:     com_uber_neuropod_Neuropod
- * Method:    nativeNew
- * Signature: (Ljava/lang/String;)J
- */
-JNIEXPORT jlong JNICALL Java_com_uber_neuropod_Neuropod_nativeNew__Ljava_lang_String_2(JNIEnv *, jclass, jstring);
-
```

Comment:
Why is this removed here?

Reply:
They are unused for a function that I have deleted. I forgot to remove this part in previous commit.